packages/web-runtime/src/pages/account.vue

```diff
@@ -20,39 +32,35 @@
       <h2 v-translate class="oc-text-bold oc-text-initial oc-mb">Account Information</h2>
 
       <dl class="account-page-info oc-flex oc-flex-wrap">
-        <div class="oc-mb oc-width-1-2@s">
-          <dt v-translate class="account-page-info-username oc-text-normal oc-text-muted">
-            Username
-          </dt>
+        <div class="account-page-info-username oc-mb oc-width-1-2@s">
+          <dt v-translate class="oc-text-normal oc-text-muted">Username</dt>
           <dd>
             {{ user.username || user.id }}
           </dd>
         </div>
-        <div v-if="user.username && user.id">
-          <dt v-translate class="account-page-info-userid oc-text-normal oc-text-muted">User ID</dt>
+        <div v-if="user.username && user.id" class="account-page-info-userid">
+          <dt v-translate class="oc-text-normal oc-text-muted">User ID</dt>
```

Comment:
selectors are moved one level up (not only to the title of the account info" but wrapping the whole div with `dt` and `dd` inside)